Unmarge pr
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Origin: upstream, https://github.com/ros2/Mimick/pull/31
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,9 +58,9 @@
   endif()
 
   include(ExternalProject)
-  set(mimick_version "de11f8377eb95f932a03707b583bf3d4ce5bd3e7")
+  set(mimick_version "90d02296025f38da2e33c67b02b7fa0c7c7d460c")
   externalproject_add(mimick-${mimick_version}
-    GIT_REPOSITORY https://github.com/ros2/Mimick.git
+    GIT_REPOSITORY https://github.com/ziyao233/Mimick.git
     GIT_TAG ${mimick_version}
     GIT_CONFIG advice.detachedHead=false
     # Suppress git update due to https://gitlab.kitware.com/cmake/cmake/-/issues/16419
