Unmarge pr
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Origin: upstream, https://github.com/ros2/Mimick/pull/31
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,7 +58,7 @@
   endif()
 
   include(ExternalProject)
-  set(mimick_version "de11f8377eb95f932a03707b583bf3d4ce5bd3e7")
+  set(mimick_version "0ad7ce1eef63d1606687a14e7b19d69e7b87bd2d")
   externalproject_add(mimick-${mimick_version}
     GIT_REPOSITORY https://github.com/ros2/Mimick.git
     GIT_TAG ${mimick_version}
